lang/typescript/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@shopify/worldwide",
+  "description": "Utilties for parsing and formatting address fields",
+  "version": "0.1.0-dev",
+  "repository": "git@github.com:Shopify/worldwide.git",
+  "author": "Shopify Inc.",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "main": "dist/index.js",
+  "types": "dist/src/index.d.ts",
+  "packageManager": "yarn@4.2.2",
```

Comment:
One small nuance that might have been missed in the original `yarn` vs `pnpm` comparison: most internal teams using yarn are still on v1 (!). That was the only supported version in shopify-build CI, see this thread for more context: https://github.com/Shopify/shopify-build/issues/4975.

Since this is an open source repository you will likely use Github Actions over Shopify Build and won't face the limitations, so this might be a non-issue. Just noting that if you run into recurring issues with yarn v4+ (like pnp & eslint) there [may not be many other internal examples to work from](https://grokt.shopify.io/results?q=packageManager+file%3Apackage.json) as some of the big apps move to pnpm instead.